todo_app/app.py

```diff
@@ -24,42 +24,52 @@ def changeItem():
         if itemTitle == "":
             flash('ERROR: No title specified for new item')
         else:
-            session_items.add_item(itemTitle)
+            trello_items.addItem(itemTitle)
```

Comment:
I like how clean a swap this is from session to trello - a good sign of choosing a good consistent interface even though the storage itself has changed!

Reply:
Thanks!